change/@internal-react-components-60cee620-62c5-430b-b711-33bc1a4fdc34.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "created error bar to display trouble shooting links for network/device permission errors",
```

Comment:
nit: Create error bar to display troubleshooting links for network/device permission errors